config/services.php

```diff
@@ -105,4 +139,19 @@ function service(string $id): ReferenceConfigurator
             service(StreamedResponseListener::class.'.inner'),
             '%env(default::RR_MODE)%',
         ]);
+
+    $services
```

Comment:
Everything related to Temporal should only be loaded if the package is installed.